README.md

```diff
@@ -22,13 +22,17 @@ The RecordLinker is a service that links records from two datasets based on a se
 Set up a Python virtual environment and install the required development dependencies:
 NOTE: Sourcing the script is recommended over simply executing the script. This allows
 the virtual environment to stay active in your shell.
+
 ```bash
 source scripts/bootstrap.sh
 ```
 
+Note: If you are running in WSL on a Windows machine, will need to run the bootstrap file directly with `.\scripts\bootstrap.sh` and then activate the virtual environment by running `.venv\bin\Activate.ps1`.
```

Comment:
@m-goggins maybe I'm getting my terminology mixed up, but did we determine its `.venv/bin/activate` on WSL and `.venv\bin\Activate.psq` on PowerShell?  Also note the slashes, I thought in WSL you use the standard Linux forward slash.

Reply:
Oh yep, you're correct! I had both terminals opened and copied from the wrong one.